internal/test/smoke/suite_base_test.go

```diff
@@ -592,6 +633,7 @@ func (s *SuiteBase) RunTestCase(testCase *TestCase) {
 	}
 
 	// WHEN
+	s.As(XRHIDProfile(testCase.Given.XRHIDProfile))
```

Comment:
Is there is a reason to use the type conversion here? It looks like that `mate test` works also with `s.As(testCase.Given.XRHIDProfile)`.